C23_WMDE_Mobile_DE_01

moiikana opened the pull request:
- first pre-campaign test for German Mobile DE

- based on last year's mobile test 14 ctrl
- changes in slides and text in both ctrl and var
- progress bar is removed in both banners

- VAR: less payment options in the form

https://phabricator.wikimedia.org/T340622

gbirke commented on desktop/templates/banner_text_var.hbs:
```diff
@@ -0,0 +1,26 @@
+	zurück.
+	Vielen Dank
+    <span style="visibility:hidden">Als ich mich entschloss, Wikipedia zu einem nicht-kommerziellen Projekt zu machen, wurde mir gesagt, ich würde dies bereuen.</span>
+    <span style="visibility:hidden"> ‒ Jimmy Wales, Wikipedia-Gründer</span>
```
Please add `class="text__paragraph--sans-serif"` to make this part identical to CTRL

moiikana committed:
Prepare campaign for C23_WMDE_Mobile_DE_01

- first pre-campaign test for German Mobile DE

- based on last year's mobile test 14 ctrl
- changes in slides and text in both ctrl and var
- progress bar is removed in both banners

https://phabricator.wikimedia.org/T340622

moiikana committed:
Implement VAR for C23_WMDE_Mobile_DE_01

- removes 3 payment types:
one of them is SmsBox, which I chose to hide via CSS,
like we do it with desktop banners for now
- add missing submit event definition to event_map.ts file

- change old IBAN to new IBAN in the footers (changed in 2023)

- disable credit card payment temporarily

https://phabricator.wikimedia.org/T340622

gbirke commented on banners/mobile/components/FullPageBanner.vue:
```diff
@@ -6,7 +6,6 @@
 			</button>
 			<div class="wmde-banner-full-info">
 				<slot name="banner-text"/>
-				<slot name="progress"/>
```
You could leave the slot in for adding it back later more easily, but removing it here probably saves a few bytes ...

gbirke reviewed (approved):
The gap in the CTRL banner (caused by the missing credit card) doesn't look nice, but the code does :)
